dGame/dComponents/InventoryComponent.cpp

```diff
@@ -994,7 +994,7 @@ void InventoryComponent::HandlePossession(Item* item) {
 	auto* destroyableComponent = mount->GetComponent<DestroyableComponent>();
 	if (destroyableComponent) {
 		destroyableComponent->SetIsSmashable(false);
-		destroyableComponent->SetIsImmune(true);
+		// TODO: Get proper immunities
```

Comment:
left over from a refactor, and it's part of the mount system which wasn't in live so my thought was "I'll lookup what the packets say later", will remove since the previous code works